agent/rpm/pbench-agent.spec.j2

```diff
@@ -39,6 +39,11 @@ Requires:  python3, python3-pip
 # RPMs for modules in requirements.txt
 Requires:  python3-bottle, python3-cffi, python3-click, python3-daemon
 Requires:  python3-jinja2, python3-redis, python3-requests, python3-sh
+
+%if 0%{?fedora} >= 36
+Requires: python3-ifaddr
+%endif
```

Comment:
No - it's only available as an RPM package on F35 and F36. Everybody else does a `pip install` to get it. That's what F36 tried to do as well, but that failed because of the PYTHONPATH problems. So I think it has to be conditional.